frontend/src/components/dashboard/aliases/AliasList.test.tsx

```diff
@@ -69,9 +69,36 @@ describe("<AliasList>", () => {
     );
   });
 
+  it("sends a request to the back-end to update the label if server-side label storage is enabled, even if the add-on is present", async () => {
+    const updateCallback = jest.fn();
+    const storeLocalLabelCallback = jest.fn();
+    LocalLabelsMock.setMockLocalLabelsOnce(
+      LocalLabelsMock.getReturnValueWithAddon([], storeLocalLabelCallback)
+    );
+    render(
+      <AliasList
+        aliases={[getMockRandomAlias()]}
+        onUpdate={updateCallback}
+        onCreate={jest.fn()}
+        onDelete={jest.fn()}
+        profile={getMockProfileData({ server_storage: true })}
+        user={{ email: "arbitrary@example.com" }}
+      />
+    );
+
+    const labelField = screen.getByRole("textbox");
+    userEvent.type(labelField, "Some label");
+    userEvent.tab();
+
+    expect(updateCallback).toHaveBeenCalledWith(expect.anything(), {
+      description: "Some label",
+    });
+    expect(storeLocalLabelCallback).not.toHaveBeenCalled();
```

Comment:
question: So this is asserting that we *DO* store the description in the local storage, but *NOT* call the function which sends the data to the server?

Reply:
Other way around: _DON'T_ store it in local storage (`storeLocalLabelCallback` is `.not.` called), _DO_ call the function which sends it to the server (`updateCallback` _is_ called).

Reply:
ah, okay got it - `updateCallback` is the one that sends to server, and `storeLocalLabelCallback` is the one that stores locally. makes sense.